src/main/java/core/basesyntax/supplier/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.supplier;
+
+import core.basesyntax.constant.Color;
+import core.basesyntax.figure.Circle;
+import core.basesyntax.figure.Figure;
+import core.basesyntax.figure.IsoscelesTrapezoid;
+import core.basesyntax.figure.Rectangle;
+import core.basesyntax.figure.RightTriangle;
+import core.basesyntax.figure.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int NUMBERS_OF_FIGURES = 5;
+    private static final int DEFAULT_RADIUS_FOR_CIRCLE = 10;
+    private static final String DEFAULT_COLOR = Color.WHITE.name();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();

```

Comment:
static vars are usually separated from the usual with a line break
```suggestion
    private static final int NUMBERS_OF_FIGURES = 5;
    private static final int DEFAULT_RADIUS_FOR_CIRCLE = 10;
    private static final String DEFAULT_COLOR = Color.WHITE.name();
    
    private final ColorSupplier colorSupplier = new ColorSupplier();
    private final Random random = new Random();

```